codespell_lib/data/dictionary.txt

```diff
@@ -16917,7 +16930,8 @@ medeival->medieval
 medevial->medieval
 medhod->method
 medhods->methods
-mediciney->mediciny
+mediciney->medicine
```

Comment:
Can you suggest medicinal here too for the same reason.

Reply:
added